### DISCUSSION TITLE
[@types/apicalypse] types use old axios version with vulnerability

### DISCUSSION THREAD

**Discussion by GerardPolloRebozado** [DefinitelyTyped/DefinitelyTyped#71115] [Issues with a @types package]
apicalypse uses axios version 0.19 wich contains a critical security vulnerability:

axios@0.19.2: Critical security vulnerability fixed in v0.21.1. For more information, see https://github.com/axios/axios/pull/3410

[a comment by an automated account is not shown]

**Comment by susam-projects** [comment]
@GerardPolloRebozado Will be fixed in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/72426
